cpg-language-python/src/main/kotlin/de/fraunhofer/aisec/cpg/frontends/python/StatementHandler.kt

```diff
@@ -84,7 +83,56 @@ class StatementHandler(frontend: PythonLanguageFrontend) :
     }
 
     /**
-     * Translates a Python (https://docs.python.org/3/library/ast.html#ast.Assert] into a
+     * Translates an [`ExceptHandler`](https://docs.python.org/3/library/ast.html#ast.ExceptHandler)
+     * to a [CatchClause].
+     *
+     * It adds all the statements to the body and will set a parameter if it exists. For the
+     * catch-all clause, we do not set the [CatchClause.parameter].
+     */
+    private fun handleExcepthandler(excepthandler: Python.AST.excepthandler): CatchClause {
+        val catchClause = newCatchClause(excepthandler)
+        catchClause.body = newBlock(excepthandler)
```

Comment:
we have a `makeBlock()`

Reply:
I will look into it.

Reply:
I already changed it and resolved the conversation ;)